projects/wvr-elements/src/lib/wvr-lib.module.ts

```diff
@@ -57,16 +73,18 @@ const components = [
   schemas: [CUSTOM_ELEMENTS_SCHEMA]
 })
 export class WvrLibModule {
-
   constructor(injector: Injector) {
-    elements.forEach(element => {
-      try {
-        const strategyFactory = new ElementZoneStrategyFactory(element.component, injector);
-        customElements.define(element.selector, createCustomElement(element.component, { injector, strategyFactory }));
-      } catch (e) {
-        // console.warn(e);
-      }
-    });
+    initializeConfig(injector.get(ConfigService))

```

Comment:
Why not just `injector.get(ConfigService).load()`? or better inject ConfigService in the constructor and call `configService.load()`?

Reply:
This issue will constitute a reworking of the configuration approach: https://github.com/orgs/TAMULib/projects/4#card-41868508 code to the effect mentioned here has already been written for this issue's implementation.